source/includes/project-membership.md

```diff
@@ -1,7 +1,7 @@
 # Project Membership
 
-##### Endpoint: `/api/v1/users/:id/project_memberships`
-##### Endpoint: `/api/v1/users/:id/project_members`
+##### Endpoint: `/api/v1/projects/:id/project_memberships`
+##### Endpoint: `/api/v1/projects/:id/project_members`
```

Comment:
Nope, just the second one. users have project_memberships, projects have project_members.

Reply:
Oh right...